src/assets/scss/_give.scss

```diff
@@ -523,6 +523,137 @@ hr.horizontal-divider {
   font-weight: 700;
 }
 
+.button-group {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+
+  .help-block {
+    padding: 10px;
+    border: 2px solid #f9b625;
+    border-radius: 2px;
+    background-color: lighten($colorCru-gold, 10%);
+    color: darken($colorCru-gold, 30%);
+  }
+
+  input[type="radio"] {
+    position: absolute;
+    cursor: pointer;
+    opacity: 0;
+  }
+
+  label {
+    display: flex;
+    flex: 1;
+    margin: 0;
+    cursor: pointer;
+  
+    .amount-box {
+      min-width: 50px;
+      align-content: center;
+      border-radius: 2px 0 0 2px;
+      margin: 0;
+      aspect-ratio: 1 / 1;
+      background-color: $colorCru-border;
+      color: inherit;
+      text-align: center;
+      transition: all 0.3s ease;
+    }
+
+    .amount-description {
+      width: 100%;
+      align-content: center;
+      padding: 5px 10px;
+      margin: 0;
+      background-color: $colorGrayscale-pebble;
+      font-weight: normal;
+    }
+  }
+
+  input[type="radio"]:hover + label {
+    .amount-description {
+      background-color: $colorGrayscale-pebble;
+    }
+  }
+
+  input[type="radio"]:focus + label {
+    .amount-box {
+      min-width: 55px;
+      background-color: $colorGrayscale-charcoal;
+      color: $colorCru-white;
+    }
+  }
+
+  input[type="radio"]:checked + label {
+    .amount-box {
+      min-width: 55px;
+      background-color: $colorGrayscale-charcoal;
+      color: $colorCru-white;
+    }
+  }
+}
+
+.custom-amount-button {
```

Comment:
Did we lose the styles that made the custom amount radio button's selected state look the same as the selected state of the other radio buttons?

<img width="1416" alt="Screenshot 2025-05-07 at 2 37 19 PM" src="https://github.com/user-attachments/assets/ddf2777d-6b0f-495d-80a1-4edaaafe35dd" />

vs

<img width="1391" alt="Screenshot 2025-05-07 at 2 38 28 PM" src="https://github.com/user-attachments/assets/b53f275b-ac25-418b-9a92-131731e97689" />